slapo/schedule.py

```diff
@@ -219,6 +224,24 @@ def __contains__(self, full_path):
                 return False
         return True
 
+    def get_params_if_tied(self, old_param, new_param):
```

Comment:
The function name seems not clear to me. Maybe "get_or_set_tied_param" or something like that.

Reply:
Fixed